hack/update-generated-crd.sh

```diff
@@ -1,11 +1,25 @@
 #!/bin/bash
 set -euo pipefail
 
-VENDORED_CRD='vendor/github.com/openshift/api/operator/v1/0000_70_dns-operator_00.crd.yaml'
-LOCAL_CRD='manifests/0000_70_dns-operator_00.crd.yaml'
-
-if [[ -z "${SKIP_COPY+1}" ]]; then
-  if ! cmp -s "$LOCAL_CRD" "$VENDORED_CRD"; then
-    cp -f "$VENDORED_CRD" "$LOCAL_CRD"
+function install_crd {
+  if [[ -z "${SKIP_COPY+1}" ]]; then
+    local SRC="$1"
+    local DST="$2"
+    if ! cmp -s "$DST" "$SRC"; then
+      cp -f "$SRC" "$DST"
+    fi
   fi
-fi
+}
+
+# Can't rely on associative arrays for old Bash versions (e.g. OSX)
+install_crd \
+  "vendor/github.com/openshift/api/operator/v1/0000_70_dns-operator_00.crd.yaml" \
+  "manifests/0000_70_dns-operator_00.crd.yaml"
+
+install_crd \
+  "vendor/github.com/openshift/api/network/v1alpha1/0000_70_dnsnameresolver_00-techpreview.crd.yaml" \
+  "manifests/0000_70_dnsnameresolver_00-techpreview.crd.yaml"
+
+install_crd \
+  "vendor/github.com/openshift/api/network/v1alpha1/0000_70_dnsnameresolver_00-customnoupgrade.crd.yaml" \
```

Comment:
Is there a soft link from https://github.com/openshift/api/tree/bec9f0f174e9decdc6d31b45cf018fd6c29f3fe2/network/v1alpha1/zz_generated.crd-manifests/0000_70_dns_00_dnsnameresolvers-CustomNoUpgrade.crd.yaml?

Reply:
As the version of openshift/api used in CDO is an older one, the soft link still resides in the vendored directory `openshift/api/network/v1alpha1` .

Reply:
Shouldn't we see these files in the vendor directory of this PR?

Reply:
Oh, I see, you have these vendor files checked in already.